src/pages/LoginSignup/LoginSignup.js

```diff
@@ -0,0 +1,28 @@
+import { lazy, Suspense, React } from 'react';
+import { Route, BrowserRouter as Switch } from 'react-router-dom';
+
+import './LoginSignup.scss';
+
+const Login = lazy(() => import('./Login'));
+const Signup = lazy(() => import('./Signup'));
+
+function LoginSignup() {
```

Comment:
we dont need this file. this should be handled in App.jsx

Reply:
Agreed, let's put our routes in `App.jsx`. Also lets not name `BrowserRouter` as `Switch` because `react-router-dom` has a component name `Switch`, rename it to `BrowserRouter as Router` to avoid confusion